vignettes/options.Rmd

```diff
@@ -30,7 +30,7 @@ table_ops <- tribble(
    ~option, ~value, ~description,
    "log.rx", "An empty R environment", "Used to store log elements during program exection",
    "log.rx.exec.env", "NULL", "The environment in which the program code is executed",
-   "[log.rx.lint]", "FALSE", "A `lintr` object for use in lint checking",
+   "[log.rx.lint]", "library_call_linter()", "A `lintr` object for use in lint checking",
```

Comment:
Nice addition. It seems like this is the main point that users would learn about where this function should be used. That's okay with me.